x-pack/legacy/plugins/apm/scripts/setup-custom-kibana-user-role.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/* eslint-disable no-console */
+/* eslint-disable import/no-extraneous-dependencies */
+
+/*
+ * This scripts will create a role that has access to kibana and assign that role to select users
+ * This makes it possible to use the existing cloud users locally
+ * Usage: node setup-custom-kibana-user-role.js --username YOUR-GITHUB-USERNAME
+ ******************************/
+const yaml = require('js-yaml');
+const axios = require('axios');
+const fs = require('fs');
+const { argv } = require('yargs');
+
+const config = yaml.safeLoad(
+  fs.readFileSync('../../../../../config/kibana.dev.yml', 'utf8')
+);
```

Comment:
This would allow the script to be run from everywhere I think:

```js
const config = yaml.safeLoad(
  fs.readFileSync(
    path.join(__filename, '../../../../../../config/kibana.dev.yml'),
    'utf8'
  )
);
```